src/popup/pages/SignMessage.vue

```diff
@@ -25,23 +25,31 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
```

Comment:
This line is not necessary. Also please organise the imports so the first ones will be JS/TS and the last the Vue components.